toptal/picasso
# feat(ProgressBar): [TEA-2569] Introduce ProgressBar component

[TEA-2569]

[BASE design](https://share.goabstract.com/4bd9b65d-c997-4c2a-a656-fcf29694cb20?mode=design&sha=ff4cdea1e7d6422f35083b4c419155d0eb92ea5a)

### Description

Introduce `ProgressBar` component to `@toptal/picasso-lab` package.

### How to test

- Run storybook locally or visit temploy
- Under `Picasso Lab`, find `ProgressBar` component
- Check if component covers all requirements from ticket description
- Check if component is implemented according [BASE design](https://share.goabstract.com/4bd9b65d-c997-4c2a-a656-fcf29694cb20?mode=design&sha=ff4cdea1e7d6422f35083b4c419155d0eb92ea5a)

### Screenshots

<img width="371" alt="Screenshot 2021-05-06 at 01 20 25" src="https://user-images.githubusercontent.com/7594216/117221291-43c73700-ae09-11eb-92a5-d34c89f03f09.png">

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[TEA-2569]: https://toptal-core.atlassian.net/browse/TEA-2569
